src/Swagger/Serializer/DocumentationNormalizer.php

```diff
@@ -508,7 +508,7 @@ private function getDefinitionSchema(string $resourceClass, ResourceMetadata $re
      */
     private function getPropertySchema(PropertyMetadata $propertyMetadata, \ArrayObject $definitions, array $serializerContext = null): \ArrayObject
     {
-        $propertySchema = new \ArrayObject();
+        $propertySchema = new \ArrayObject($propertyMetadata->getAttributes()['swagger_context'] ?? []);
```

Comment:
Isn't `$propertyMetadata->getAttributes()['swagger_context'] ?? [] ` enough?

Reply:
Not if you want to return an `ArrayObject` instance on line 522 https://github.com/api-platform/core/pull/1466/files#diff-32d63273c86a7e4c14c3fa6bb4131194R522